content/_data/changelogs/weekly.yml

```diff
@@ -4345,6 +4345,47 @@
     # pull: 3986 (Making ConfidentialKey instances safe to keep in static fields even across JenkinsRule starts)
     # pull: 3990 (Bump to more recent tip of ATH)
     # pull: 3995 (Add section for agent to master configuration)
+- version: 2.176
+  date: 2019-05-05
+  changes:
+    - type: major bug
+      pull: 4011
+      issue: 57244
+      message: |-
+        Some plugins would fail with an error related to serialization of <code>FilePath</code>s.
+        This is now downgraded to a warning.
+        Plugin updates should still be applied to correct the underlying mistake.
+        (regression in 2.175)
+    - type: rfe
+      pull: 3686
+      issue: 11888
+      message: |-
+        Add <code>stop-job</code> CLI command which allows aborting builds.
+    - type: rfe
+      pull: 4005
+      references:
+        - issue: 51004
+        - issue: 57107
+        - issue: 46515
+        - url: https://github.com/jenkinsci/remoting/blob/master/CHANGELOG.md#330
+          title: full changelog
+      message: |-
+        Update Remoting from 3.29 to 3.30 to fix a few minor issues.
+    - type: rfe
+      pull: 4008
+      message: |-
+        Migrate all Chinese localization resources into <a href="https://plugins.jenkins.io/localization-zh-cn">Localization: Chinese (Simplified) plugin</a>.
+    - type: bug
+      pull: 4004
+      issue: 57273
+      message: |-
+        <code>NodeListener#onCreated</code> was called when <code>Jenkins#addNode</code> or <code>Nodes#addNode</code> actually replaced an existing node.
+    - type: rfe
+      pull: 4006
+      message: |-
+        Developer: include generated source files of CLI module in the classpath.
+    # pull: 4012 Run.keepLog-related names were misleading
+    # pull: 3936 [JENKINS-56499] Remove synchronized on method updateUsageForId for API token usage stat
```

Comment:
I would add it to the changelog as rfe. It is a performance improvement which worth mentioning imho (mostly because it comes from an external contributor)